templates/acm-certificate.yaml

```diff
@@ -4,95 +4,34 @@ Description: ACFS3 - Certificate creation
 Parameters:
   DomainName:
     Type: String
-  Region:
-    Type: String
-    Default: 'us-east-1'
-  CFNCustomProvider:
-    Type: String
-  CopyFunction:
-    Type: String
   SubDomain:
     Type: String
   CreateApex:
     Type: String
+  HostedZoneId:
+    Type: String
 
 Conditions:
   CreateApexConfig:  !Equals 
     - !Ref CreateApex
     - 'yes'
 
 Resources:
-  CopyCustomResource:
-    Type: "AWS::CloudFormation::CustomResource"
-    Properties:
-      ServiceToken: !Ref CopyFunction
-
   Certificate:
-    Type: Custom::Certificate
-    Properties:
+    Type: "AWS::CertificateManager::Certificate"
```

Comment:
None of the other types in the template are quoted, might want to remove them to keep it consistent.

Reply:
Good point. Fixed